inference-engine/src/readers/ir_reader/ie_ir_reader.cpp

```diff
@@ -55,7 +55,7 @@ INFERENCE_PLUGIN_API(StatusCode) InferenceEngine::CreateReader(IReader*& reader,
         reader = new IRReader();
         return OK;
     }
-    catch (std::exception &ex) {
+    catch (...) {
```

Comment:
Why did you change `std::exception` to any exception?

Reply:
'ex' is never used. Code: LV_UNUSED.GEN

Reply:
can we have `catch (std::exception &)` ? Without actual variable, but type of exception is still the same.

Reply:
fixed